grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -702,12 +702,14 @@ class DataService {
                 } else if (element.defaultValue && !value) {
                     value = element.defaultValue
                 }
-                properties[fieldName] = value ?: ""
+                // We can't just check the truthiness of the actualValue, because the false boolean would be evaluated to an empty string
```

Comment:
"actualValue" -> "value" in these two comments

Reply:
Thanks, I copied it from the other file and forgot to change it